syncer/job.go

```diff
@@ -111,10 +112,21 @@ func newDDLJob(ddlInfo *shardingDDLInfo, ddls []string, location, cmdLocation bi
 	}
 
 	if ddlInfo != nil {
-		j.sourceSchema = ddlInfo.tableNames[0][0].Schema
-		j.sourceTable = ddlInfo.tableNames[0][0].Name
+		j.sourceSchemas = []string{ddlInfo.tableNames[0][0].Schema}
+		j.sourceTables = []string{ddlInfo.tableNames[0][0].Name}
 		j.targetSchema = ddlInfo.tableNames[1][0].Schema
 		j.targetTable = ddlInfo.tableNames[1][0].Name
+	} else if sourceTbls != nil {
```

Comment:
in which case `ddlInfo` is nil and `sourceTbls` is not nil? can you add a example

Reply:
updated in https://github.com/pingcap/dm/pull/587/commits/2733185c2787894f9104e20aab935ac154092e4b